variables.tf

```diff
@@ -4,6 +4,12 @@ variable "instance_type" {
   default     = "t3.large"
 }
 
+variable "host_name" {
+  description = "Middle name for the host, default is random"
```

Comment:
MIddle?

Reply:
first part of the name is always MM

Reply:
Ok, please use `mm` (lowercase). Looks good otherwise